app/main/views.py

```diff
@@ -146,13 +249,13 @@ def pagination_links(pagination, endpoint, args):
     ]
 
 
-def get_json_from_request():
-    if request.content_type != 'application/json':
+def get_json_from_request(key):
```

Comment:
I'm not sure this is the right pattern. It means that we do the first two checks repeatedly for each invocation (that may also include JSON deserialisation). Maybe pass keys in as an array and then use the returned dictionary? (I'm not convinced this is a better pattern).

```python
data = get_json_from_request(['updater', 'serviceUpdate'])
validate_updater_json_or_400(data['updater'])
service_update = data['service_update']
```